build.sh

```diff
@@ -0,0 +1,9 @@
+#docker buildx build --platform linux/arm64 -t meshlake/datahub-ingestion:arm64 --push -f ./docker/datahub-ingestion/Dockerfile . 
```

Comment:
Please remove this file. Why would the OSS have `meshlake/datahub-ingestion` in the source code? This should be in your local system